policies/queries/mq/brokers_unused.sql

```diff
@@ -0,0 +1,14 @@
+insert into aws_policy_results
+with broker_user as (select distinct broker_cq_id from aws_mq_broker_users)
+select :'execution_time'                           as execution_time,
+       :'framework'                                as framework,
+       :'check_id'                                 as check_id,
+       'Unused ActiveMQ brokers should be removed' as title,
+       broker.account_id,
+       broker.arn                                  as resource_id,
+       case
+           when broker_user.broker_cq_id is null then 'fail'
+           else 'pass'
+           end                                     as status
+from aws_mq_brokers broker
+         left join broker_user on broker_user.broker_cq_id = aws_mq_brokers.cq_id
```

Comment:
When a user uses LDAP for authentication do users still get created and appear in the `broker_user` table?

Reply:
no, removing the check